BridgeCareApp/BridgeCare/DataAccessLayer/LatLonSection.cs

```diff
@@ -0,0 +1,42 @@
+﻿using BridgeCare.ApplicationLog;
+using BridgeCare.Interfaces;
+using BridgeCare.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+
+namespace BridgeCare.DataAccessLayer
+{
+    public class LatLonSection : ILatLonSection
+    {
+        public LatLonSection() { }
+        public IQueryable<LatLonSectionModel> GetLatLon(int NetworkId, BridgeCareContext db)
+        {
+            IQueryable<LatLonSectionModel> rawQueryForData = null;
+
+            // bug in program: Long is null in rolled up DB even when the source DB has valid data, including isnull 
```

Comment:
Bug descriptions don't belong here in the comments. Please create a bug branch and describe the issue there.